src/components/Data/Browse/IndexBranch.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <div v-bind:class="{ 'open': open }">
+    <i v-if="collectionCount(index)" class="fa fa-caret-right tree-toggle" aria-hidden="true" @click="toggleBranch()"></i>
+    <!-- v-link="{name: 'DataIndex', params: {index: index.name}}" -->
+    <a class="tree-item truncate"
+       v-bind:class="{ 'active': isIndexActive($route, index.name) }">
```

Comment:
Could we say we don't use v-bind: and v-on: but shortcut instead? like :class? (just like for @click in line 3)